src/App.js

```diff
@@ -1,5 +1,123 @@
+import Background from "./assets/Background.svg";
+import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
+
+async function post(url, body) {
+  const response = await fetch(process.env.REACT_APP_API_URL + url, {
+    method: "POST",
+    body: JSON.stringify(body),
+    headers: {
+      "Content-type": "application/json; charset=UTF-8",
+    },
+  });
+  const json = await response.json();
+  return json;
+}
+
 function App() {
-  return <h1 className="text-3xl font-bold underline">Hello world!</h1>;
+  const navigate = useNavigate();
+
+  const handleSubmit = async (event) => {
```

Comment:
I commented this in previous PRs, but the correct way to get data from a form is FormData (https://github.com/Subvisual-Academy/space/pull/11/#discussion_r1259528137). Also, the inputs in the form need `name` attributes

Reply:
Yes, Tiago had changed that, my code was outdated in some files, but I've already solved that. TY